watershed_workflow/hydrography.py

```diff
@@ -487,23 +487,20 @@ def prune_by_segment_length(tree, prune_tol=10):
         if leaf.segment.length < prune_tol:
             logging.info("  ...cleaned leaf segment of length: %g at centroid %r" %
                          (leaf.segment.length, leaf.segment.centroid.coords[0]))
-            if 'area' in leaf.properties and 'area' in leaf.parent.properties:
-                leaf.parent.properties['area'] += leaf.properties['area']
-            leaf.remove()
+            leaf.removePreserveAreas()
```

Comment:
This line throws the error _"'River' object has no attribute 'removePreserveAreas'"_ in some runs. The error can be reproduced by running _mesh_mixed_element_TaylorPark_upstream.ipynb_ in WW examples. This tends to happen when _merge_tol_ is provided by the user.

Reply:
fixed